scripts/fast-reboot

```diff
@@ -63,6 +63,12 @@ then
   systemctl stop "$service_name"
 fi
 
+# Stop kernel modules for Nephos platform
+if [ "$sonic_asic_type" = 'nephos' ];
```

Comment:
can you use [[  ]] instead of [ ]?

https://stackoverflow.com/questions/3427872/whats-the-difference-between-and-in-bash